v2/pkg/core/inputs/hybrid/hmap.go

```diff
@@ -134,3 +144,27 @@ func (i *Input) Scan(callback func(value string)) {
 		i.hostMap.Scan(callbackFunc)
 	}
 }
+
+// expandCIDRInputValue expands CIDR and stores expanded IPs
+func (i *Input) expandCIDRInputValue(value string) {
+	ips, _ := mapcidr.IPAddresses(value)
```

Comment:
Probably `mapcidr.IPAddressesAsStream(value)` is preferred since large network ranges would consume a lot of memory.

Reply:
Updated the function to use `mapcidr.IPAddressesAsStream(value)`